docs/source/release/v3.13.0/muon.rst

```diff
@@ -30,6 +30,7 @@ New
 ###
 - :ref:`ConvertFitFunctionForMuonTFAsymmetry <algm-ConvertFitFunctionForMuonTFAsymmetry>` has been added to help convert fitting functions for TF asymmetry fitting.
 
+
```

Comment:
need to delete this line as it will cause errors

Reply:
What errors would it cause? The builds all seemed to work ok. Should all double white lines be removed in these files?

Reply:
you need a single line after the bullets